students/DanCornutt/lessons/lesson08/assignment/inventory.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+"""HW8 Functional Methods"""
+import os.path
+import csv
+from functools import partial
+
+
+def add_furniture(
+    invoice_file, customer_name, item_code, item_description,
+    item_monthly_price
+):
+    """
+    This function will create invoice_file if it doesn’t\
+    exist or append a new line to it if it does.
+        param1: invoice file
+        param2: customer name
+        param3: item code
+        param4: item description
+        param5: item monthly price
+    """
+    if os.path.isfile(invoice_file):
+        open_type = 'a'
+    else:
+        open_type = 'w'
+    with open(invoice_file, open_type, newline='') as csv_file:
+        writer = csv.writer(csv_file, delimiter=',')
+        writer.writerow([customer_name, item_code, item_description,
+                         item_monthly_price])
```

Comment:
Even if you open the file using "a", it will create the file, so you can just always open as a.

Reply:
Oh nice, good to know!